src/app/zap-templates/af-structs.zapt

```diff
@@ -0,0 +1,20 @@
+{{chip_header}}
+
+// Enclosing macro to prevent multiple inclusion
+#ifndef __ZAP_AF_STRUCTS__
```

Comment:
`#pragma once`?  And similar for other headers?

Reply:
Done